OpenRA.Game/Map/CellLayer.cs

```diff
@@ -122,6 +122,12 @@ IEnumerator IEnumerable.GetEnumerator()
 		{
 			return GetEnumerator();
 		}
+
+		public bool Contains(MPos uv)
+		{
+			return new Rectangle(0, 0, Size.Width, Size.Height)
```

Comment:
Is it worth it to store that rectangle somewhere so you save yourself the object creation for every cell in every tick?